moonstone/plot/graphs/base.py

```diff
@@ -62,7 +64,10 @@ def _handle_plotting_options_plotly(self, fig, plotting_options: dict):
             getattr(fig, updater)(plotting_options[option])
         return fig
 
-    def _handle_output_plotly(self, fig, show, output_file):
+    def _handle_output_plotly(self, fig, show: bool, output_file: str, log_scale: bool = False):
+        if log_scale:
```

Comment:
log_scale could have been passed in plotting_options

- For example, in counts.py, we use : 

plotting_options = add_x_to_plotting_options(plotting_options, 'layout', 'yaxis_type', 'log')

Reply:
done in last commit @AgnesBaud